### PR TITLE
#68 updating travis

### PR DESCRIPTION
**Pull request by lorientedev**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change
Updated ansible version to 2.8.8
Installed pipenv in travis, installed dependencies and run molecule test with it.

### CODE REVIEW

File: molecule/default/group_vars/airflow/main.yml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 ---
-airflow_fernet_key: xKy13nPFfDflJ0DYGVTwf_DEmbItfURHlEDxrt-bBQw=
\ No newline at end of file
+airflow_fernet_key: xKy13nPFfDflJ0DYGVTwf_DEmbItfURHlEDxrt-bBQw=
```
Comment (vicsufer) [inline molecule/default/group_vars/airflow/main.yml:2]: If this is a secret it should not be in the role, you can use a placeholder here like `<AIRFLOW_FERNET_KEY>`, this value would be overrided later on the playbook.

**Review by vicsufer — CHANGES_REQUESTED** [review]
Can you update CHANGELOG.md?

File: molecule/default/group_vars/airflow/main.yml (hunk 1 above)
Comment (vicsufer) [reply molecule/default/group_vars/airflow/main.yml:2]: Nevermind, it's molecule testcase my bad.

**Review by xtianae7 — APPROVED** [review]

File: test-requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
-molecule==2.20.1
-docker==3.7.2
\ No newline at end of file
+molecule==3.0.1
+docker==4.1.0
+ansible-lint==4.2.0
```
Comment (xtianae7) [inline test-requirements.txt:3]: you can also define here your ansible version. this way, you wont have to install separately ansible from the rest of the requirements on travis. (and its also easier to find for future changes and local tests)

File: .travis.yml [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
+  - ansible_version: 2.8.8
 
 services:
   - docker
 
 install:
-  - pip install ansible==${ansible_version}
-  - pip install -r test-requirements.txt
+  - pip3 install pipenv
+  - pipenv install ansible==${ansible_version}
```
Comment (xtianae7) [inline .travis.yml:14]: if you add ansibles version on your requirements file, you can omit this line.

File: meta/main.yml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 galaxy_info:
   company: Idealista S.A.U.
+  author: Idealista S.A.U.
   description: Apache Airflow role
   min_ansible_version: 2.4.5.0
```
Comment (xtianae7) [inline meta/main.yml:6]: update metadata

**Review by xtianae7 — COMMENTED** [review]
Also update readme please :)

File: molecule/default/group_vars/airflow/main.yml (hunk 1 above)
Comment (lorientedev) [reply molecule/default/group_vars/airflow/main.yml:2]: Is a default fernet key to role test works, in the playbook this key is in a vault.
If I put the value as a placeholder how can specify the key for travis?

File: .travis.yml (hunk 3 above)
Comment (frantsao) [reply .travis.yml:14]: In fact, in order to do more reproducible tests, a better workflow would be:
1. update test-requirements
2. generate Pipfile and Pipfile.lock (i.e. running pipenv install -r test-requirements.txt)
3. commit Pipfile and Pipfile.lock
So the sequence in travis would be:
install:
  - pip install  pipenv
  - pipenv sync

File: molecule/default/group_vars/airflow/main.yml (hunk 1 above)
Comment (vicsufer) [reply molecule/default/group_vars/airflow/main.yml:2]: It's okay as it is, I haven't seen before that it is for a test scenario.

**Review by vicsufer — APPROVED** [review]

File: Pipfile [hunk 5]
```diff
@@ -0,0 +1,16 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+cookiecutter = "*"
```
Comment (blalop) [inline Pipfile:9]: Is cookiecutter still being used?

**Review by blalop — APPROVED** [review]

File: Pipfile (hunk 5 above)
Comment (lorientedev) [reply Pipfile:9]: Not really sorry, i change it

**Review by xtianae7 — APPROVED** [review]

**Review by caldito — APPROVED** [review]

**Pull request merged**